src/main/target/common_hardware.c

```diff
@@ -167,7 +167,7 @@
         #define LIS3MDL_I2C_BUS MAG_I2C_BUS
     #endif
     BUSDEV_REGISTER_I2C(busdev_lis3mdl,     DEVHW_LIS3MDL,      LIS3MDL_I2C_BUS,    0x1E,               NONE,           DEVFLAGS_NONE);
-//  BUSDEV_REGISTER_I2C(busdev_lis3mdl,     DEVHW_LIS3MDL,      LIS3MDL_I2C_BUS,    0x1C,               NONE,           DEVFLAGS_NONE);
+    BUSDEV_REGISTER_I2C(busdev_lis3mdl_a2,  DEVHW_LIS3MDL_A2,   LIS3MDL_I2C_BUS,    0x1C,               NONE,           DEVFLAGS_NONE);
```

Comment:
Creating a second bus device works fine, the mag is detected with either address.  It just doesn't seem very graceful.  Any other ideas?